sql/parser/keywords.go

```diff
@@ -110,6 +110,8 @@ var keywords = map[string]int{
 	"DATABASES":         DATABASES,
 	"DATE":              DATE,
 	"DAY":               DAY,
+	"DAYOFWEEK":         DAYOFWEEK,
+	"DAYOFYEAR":         DAYOFYEAR,
```

Comment:
FYI, I'm pretty sure you don't have to add all of these here and you don't have to add them to `sql.ly` either. The `extract_arg` rule accepts `IDENT` which will capture all of these strings.